docs/how-to/build-generator-to-file-pipeline.mdx

```diff
@@ -0,0 +1,172 @@
+---
+title: "How to build a generator-to-file pipeline"
+sidebar_position: 0
+---
+
+In this guide, we'll learn how to build a basic Conduit pipeline. It will use
+two built-in connectors:
+
+- A generator source (a source connector that generates random data). Our
+  pipeline will generate imaginary employee data.
+- A logging destination (a destination that simply logs all incoming records).
+
+As you'll see, all it takes to accomplish this is to write
+a [pipeline configuration file](/docs/pipeline-configuration-files/getting-started)!
+
+The steps below will guide you through installing Conduit and gradually building
+the pipeline. If you're in a hurry, you can skip ahead
+and [run the pipeline](#step-6-run-the-pipeline) immediately.
+
+## Step 1: Install Conduit
+
+Download
+the [latest Conduit release](https://github.com/ConduitIO/conduit/releases/latest)
+to a directory of your choice. In this guide, the directory will be
+`~/conduit-playground`.
+
+## Step 2: Create a pipeline configuration file
+
+Pipelines can be created through YAML configuration files, that are, by default, put into a directory called `pipelines`.
+
+With the following command, we'll create the `pipelines` directory and a file that will contain our pipeline's configuration:
+
+```bash
+mkdir pipelines && touch pipelines/generator-to-file.yaml
+```
+The directory layout should be as below:
+
+```console
+~/conduit-playground
+├── conduit
+└── pipelines
+    └── generator-to-file.yaml
+```
+
+
+## Step 3: The pipeline's basics
+
+Next, write the following to `generator-to-file.yaml`:
+
+```yaml
+version: "2.2"
+pipelines:
+    - id: pipeline1
+      # Tells Conduit to run the pipeline automatically
+      status: running
```

Comment:
I'm thinking if it would make sense to make this the default. As a user, if I create a pipeline config file, I'd expect Conduit to load it and run it. Only if I say "status: stopped" it should be stopped. WDYT?